recipes/stanalyzer/meta.yaml

```
@@ -0,0 +1,71 @@
+{%- set tag = "1.0.0dev1" -%}
+{%- set version = tag|replace('-', '_') -%}
```

Minor nit - it's probably simpler to just define `python_min` in the `meta.yaml`:
```suggestion
{%- set tag = "1.0.0dev1" -%}
{%- set version = tag|replace('-', '_') -%}
{% set python_min = "3.10" %}
```

---

In that case, I suppose I should remove the corresponding entry from `conda_build_config.yaml`, but as I want to get this moving, I don't care if it happens before or after.